go.mod

```diff
@@ -38,3 +41,5 @@ require (
 	k8s.io/kube-openapi v0.0.0-20190225204428-d50a959ae76a // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
+
+replace github.com/josephburnett/sk-plugin => ../sk-plugin
```

Comment:
Hmm, is this line correct? Looks weird, what's ../sk-plugin? Isn't it some debug leftover?

Reply:
Currently this line is essential to be able to build Skenario. Maybe it is worth creating a separate task of removing this line.